open_event/views/views.py

```diff
@@ -13,8 +13,12 @@
 from ..helpers.object_formatter import ObjectFormatter
 from flask import Blueprint
 from flask.ext.autodoc import Autodoc
+from icalendar import Calendar,Event
 
 auto=Autodoc()
+cal=Calendar()
+event=Event()
+track=Event()
```

Comment:
Unnecessary global variables are not good practice. Why not declare them in the functions that use them? And why not use icalendar.Event() so you don't have to change already existing code (which there is more of) that references Event?